web/sites/default/settings.php

```diff
@@ -10,7 +10,7 @@
 
 // Isolate any subsequently-created variables from the global scope by
 // including settings files in a self-executing anonymous function.
-(function () use (&$app_root, &$site_path, &$class_loader, &$config_directories, &$config, &$settings, &$databases) {
+(function () use (&$app_root, &$site_path, &$class_loader, &$config_directories, &$config, &$settings, &$databases, &$env) {
```

Comment:
Since `$env` isn't defined in the outer context, it will be getting passed in as NULL. The variable gets defined in `settings.common.php` and is subsequently available to all the code that follows (see [include()#1](https://secure.php.net/manual/en/function.include.php#example-122). To make this a little more apparent, I did think about defining `$env` at the top of this anonymous function in `settings.php` instead of at the top of `settings.common.php`. If you'd rather see it that way, it's good with me! :)

Reply:
As long as it's okay dokay, then good good!